config/development/kustomization.yaml

```diff
@@ -5,11 +5,11 @@ patchesStrategicMerge:
 - test_certs.yaml
 - backup_credentials.yaml
 - debug_logs.yaml
+- images.yaml
 
 resources:
 - ../deployment
 - ../minio
-
```

Comment:
Is there a reason to remove the newline?

Reply:
That was supposed to be part of a larger change that got missed in the initial commit.

Reply:
Ah, I see now that the change I was thinking of isn't going to work, so I'll fix this.